docs/workflow_solutions/creating_sandbox_apps.md

```diff
@@ -0,0 +1,120 @@
+
+    ![Help button menu options](images/cheaha_help_button.png)
+
+1. When the page reloads, you should see a new "Develop" tab have successfully enabled Sandbox for your environment, and you should see a new "Develop" tab.
+
+    ![Develop button on Cheaha](images/cheaha_develop_button.png)
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    This is a one time setup, and would not need to be repeated for subsequent apps you would like to create.
+<!-- markdownlint-enable MD046 -->
+
+## Building your first dev app
+
+Now that Sandbox is set up on your account, we can create an App. The instructions below detail how to create your first example dev app via a container:
+
+### FSL
+
+This example using the FSL (FMRIB Software Library) shows how to create a simple Sandbox App on Cheaha.
+
+1. Under "Develop" tab , click on "My SandBox Apps (Development)"
+
+    ![Develop button options on Cheaha](images/cheaha_develop_sandbox.png)
+
+1. Clicking the "New App" button, should direct you to a new page giving you an option to Clone an Existing App.
+
+    ![New App button to Clone an Existing App](images/sandbox_new_app.png)
+
+1. Now we are going to import an already existing git repo for an app, by clicking on the "Clone Existing App". A form (shown below) requiring your input will display. For the FSL App we will use a template we already created (you can use this template to prepare your app as well).
+
+    ![Form for new Sandbox App](images/sandbox_new_app_form.png)
+
+    We will add the following values in the form:
+
+    **Directory Name:** FSL
+
+    **Git Remote:** <https://gitlab.rc.uab.edu/rc/bc_uab_igv.git> and then click "**Submit**"
+
+    When this is done successfully, you should see the below screen also showing your new FSL Sandbox App.
+    ![New Sandbox App](images/new_fsl_sandbox_app.png)
+
+    <!-- markdownlint-disable MD046 -->
+    !!! note
+
+        You can also check the option "Create a new Git Project from this?", about creating a new git repo, if you want to use it to maintain your own instance of the app. In this tutorial, we would not be selecting this option.
+    <!-- markdownlint-enable MD046 -->
+
+1. Now click on "Files" to take a look at the files for the app (local git repo working directory), that you just cloned. We will modify some of the files, as you will notice the cloned app, provides a context and environment not relevant to FSL.
+
+    ![File list for Sandbox App](images/file_sandbox_app.png)
+
+    1. We first modify "manifest.yml" file (highlight the file, and click on edit) to change metadata for the sandbox app.
+       Change the variables appropriately. In this case we will change the following:
+
+        ![Metadata contained in manifest.yml file](images/manifest_yml_sandbox.png)
+
+    1. Next, select "form.yml.erb" and click on "Edit" . Uncomment version section in the file, and change that section as follows:
+  version:
```

Comment:
This indentation shouldn't be allowed by the linter. I need to check on that...

Looks like this is not yet covered by MarkdownLint. This is called a "Lazy Continuation Line" and we want to avoid it. There is an issue in their tracker to eventually add a rule to cover this case: https://github.com/DavidAnson/markdownlint/issues/977

Reply:
Rework this to avoid the continuation line.

Reply:
I took this out completely and just and referenced the `YAML` file for the appropriate section.